include/Zydis/SharedTypes.h

```diff
@@ -47,6 +47,7 @@ extern "C" {
 /* ---------------------------------------------------------------------------------------------- */
 
 #define ZYDIS_MAX_INSTRUCTION_LENGTH    15
+#define ZYDIS_MAX_DISP_IMM_SIZE         32
```

Comment:
This should be `64` to support `mov{abs}` and `jmpabs` (as soon as APX is merged).